docker/test/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '2.1'
+
+networks:
+  iam-net-test:
+    driver: bridge
+    name: "iam-net-test"
+
+services:
+  iam-server-test:
+    build:
+      context: ../../
+      dockerfile: docker/development/Dockerfile
```

Comment:
This points to the development Dockerfile instead of tests.

Reply:
Yes, they do. They are the same configuration.